app/src/main/java/de/tum/in/tumcampusapp/component/tumui/lectures/adapter/LectureAppointmentsListAdapter.java

```diff
@@ -74,50 +70,44 @@ public View getView(int position, View view, ViewGroup parent) {
         }
 
-        LectureAppointmentsRow lvItem = appointmentList.get(position);
+        LectureAppointment lvItem = appointmentList.get(position);
 
         // only show if lecture has a title and enough info
         if (lvItem != null) {
-            holder.tvTerminOrt.setText(lvItem.getOrt());
-            StringBuilder line2 = new StringBuilder(lvItem.getArt());
-            // only show betreff if available
-            if (!lvItem.getTermin_betreff()
-                       .isEmpty()) {
-                line2.append(" - ")
-                     .append(lvItem.getTermin_betreff());
+            holder.tvTerminOrt.setText(lvItem.getLocation());
+            StringBuilder line2 = new StringBuilder(lvItem.getType());
+
+            String title = lvItem.getTitle();
+
+            if (title != null && !title.isEmpty()) {
+                line2.append(" - ").append(lvItem.getTitle());
             }
             holder.tvTerminBetreff.setText(line2.toString());
 
-            try {
-                DateTime start = formatter.parseDateTime(lvItem.getBeginn_datum_zeitpunkt());
-                DateTime end = formatter.parseDateTime(lvItem.getEnde_datum_zeitpunkt());
-
-                // output if same day: we only show the date once
-                StringBuilder output = new StringBuilder();
-                if (DateTimeUtils.INSTANCE.isSameDay(start, end)) {
-                    output.append(startDateOutput.print(start))
-                          .append(" - ")
-                          .append(endHoursOutput.print(end));
-                } else {
-                    // show it normally
-                    output.append(startDateOutput.print(start))
-                          .append(" - ")
-                          .append(endDateOutput.print(end));
-                }
-
-                // grey it, if in past
-                if (start.isBeforeNow()) {
-                    output.insert(0, "<font color=\"#444444\">");
-                    output.append("</font>");
-                }
-
-                holder.tvTerminZeit.setText(Utils.fromHtml(output.toString()));
-
-            } catch (IllegalArgumentException e) {
-                holder.tvTerminZeit.setText(String.format("%s - %s",
-                                                          lvItem.getBeginn_datum_zeitpunkt(), lvItem.getEnde_datum_zeitpunkt()));
+            DateTime start = lvItem.getStartTime();
+            DateTime end = lvItem.getEndTime();
+
+            // output if same day: we only show the date once
+            StringBuilder output = new StringBuilder();
+            if (DateTimeUtils.INSTANCE.isSameDay(start, end)) {
+                output.append(startDateOutput.print(start))
+                        .append(" - ")
+                        .append(endHoursOutput.print(end));
+            } else {
+                // show it normally
+                output.append(startDateOutput.print(start))
+                        .append("–")
```

Comment:
This is an en-dash, while we used minus signs otherwise...
I think we should use en-dashes everywhere?